util/manifests.go

```diff
@@ -30,6 +30,7 @@ func ApplyManifests(m []byte, client kubernetes.Interface) error {
 		return fmt.Errorf("error parsing manifests: %v", err)
 	}
 	for i, m := range manifests {
+		var err2 error
```

Comment:
Wouldn't naming it manifestErr would make it a little more readable?

Reply:
It might. I don't think it matters that much - everything in here is a manifest error, just a question of if it is an error from getting the resource or update/apply the resource.

Reply:
I suppose we never would really name the first `err` anything related to the types of errors it may hold.

Reply:
I guess we could rename the error inside each `case` statement as `getErr`, and then move `err2` back to `err`, or similar.

But does this really buy us anything?

Personally, I dislike the massive repetitiveness of the switch statement, but the `intf` in each `case` statement is something else, something that has `.Update(arg)` and `.Create(arg)` and `.Get(string, metva1.GetOptions)`, but the `arg` is different for each, so I found no way to abstract it all out.